dodo/util.py

```diff
@@ -63,13 +63,16 @@ def w3m_html2text(s: str) -> str:
     :param s: an HTML input string
     :returns: plain text representation of the HTML
     """
-
-    (fd, file) = tempfile.mkstemp(suffix='.html')
-    with os.fdopen(fd, 'w') as f:
-        f.write(s)
-    p = subprocess.run(['w3m', '-O', 'utf8', '-dump', file],
-            stdout=subprocess.PIPE, encoding='utf8')
-    os.remove(file)
+    try:
+        p = subprocess.run(
+            ["w3m", "-T", "text/html", "-O", "utf8", "-dump"],
+            stdout=subprocess.PIPE,
+            encoding="utf8",
+            input=s,
+            check=True,
+        )
+    except (OSError, subprocess.CalledProcessError) as e:
+        return f"dodo w3m error: {e}"
```

Comment:
suggestion: maybe dump out `e.stderr` there? You'd likely need to adjust the arguments to `run()`, but AFAICT the exception text only gives the error code, which doesn't help much unless you happen to have launched dodo in a terminal.

Reply:
I'm somewhat torn on this. Right now, unfortunately a lot of error conditions just dump details to stderr, so I got used to running it from a terminal. 

In most of those cases, right now we don't have a nice place in the UI where we could show multi-line text, but yeah, here we would at least have some nice space for it.

IMHO the optimal solution would be to have some sort of debug logging console in the Dodo GUI for CLI output, but obviously that comes with its own complexity...